launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the LoadComposableNodes action."""
+
+import logging
+from typing import List
+from typing import Optional
+
+import composition_interfaces.srv
+
+from launch.action import Action
+from launch.actions import RegisterEventHandler
+from launch.event_handlers.on_process_start import OnProcessStart
+from launch.events.process import ProcessStarted
+from launch.launch_context import LaunchContext
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+from .composable_node_container import ComposableNodeContainer
+
+from ..descriptions import ComposableNode
+from ..utilities import evaluate_parameters
+from ..utilities import to_parameters_list
+
+_logger = logging.getLogger(name='launch_ros')
+
+
+class LoadComposableNodes(Action):
+    """Action that loads composable ROS nodes into a running container."""
+
+    def __init__(
+        self,
+        *,
+        composable_nodes_description: List[ComposableNode],
+        target_container: ComposableNodeContainer,
+        in_parallel: Optional[SomeSubstitutionsType] = None,
+        **kwargs,
+    ) -> None:
+        """
+        Construct a LoadComposableNodes action.
+
+        The container node is expected to provide a `~/_container/load` service for loading
```

Comment:
did we have somewhere that defined this service name?  I hadn't been using it, but will update accordingly.

Reply:
ros2/design#206 has `~/_containter/load_node` instead

Reply:
My bad, I'll fix it.

Reply:
Fixed in https://github.com/ros2/launch/pull/197/commits/b28bf50a56c63124d081626d4000b82466bc1c08.